.travis.yml

```diff
@@ -194,9 +194,8 @@ matrix:
     
     ### Extended Tests ###
     - &extended-vm
-      stage: "Extended"
-      name: "psa autogen"
-      env: NAME=psa-autogen
+      name: "events"
```

Comment:
The events test previously ran under `stage: "Extended"`. This should probably come back.

Reply:
Fixed.